src/docs/user_ai.md

```diff
@@ -262,7 +262,7 @@ Here is the updated **MCP Integration** section with the requested changes:
 
 ## MCP Integration
 
-The Theia IDE now supports an with the Model Context Protocol (MCP), enabling users to configure and utilize external services in their AI workflows. 
+The Theia IDE now supports integration with the Model Context Protocol (MCP), enabling users to configure and utilize external services in their AI workflows. 
```

Comment:
Bit awkward. Maybe

> now supports the Model Context Protocol

> now integrates the Model Context Protocol

> now supports model integration via the Model Context Protocol

Not sure exactly which is most appropriate for our relationship with the protocol, but those would sound a bit more natural.

Reply:
also removed the "now"